src/PrefixHeuristicEvaluatorMain.cpp

```diff
@@ -34,7 +34,7 @@ int main(int argc, char** argv) {
     exit(1);
   }
 
-  for (const auto& p : calculatePrefixes(argv[1], 127, 1)) {
+  for (const auto& p : calculatePrefixes(argv[1], 127, 1, true)) {
```

Comment:
I think this should use the `NUM_COMPRESSION_PREFIXES` constant so that it always matches the encoding actually in use